config/config_test.go

```diff
@@ -0,0 +1,139 @@
+package config
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func emptyConfig() *Config {
+	return &Config{ResourceType{FilterRule{}, FilterRule{}}}
+}
+
+func TestConfig_Garbage(t *testing.T) {
+	configFilePath := "./mocks/garbage.yaml"
+	configObj, err := GetConfig(configFilePath)
+
+	require.NoError(t, err)
+
+	if !reflect.DeepEqual(configObj, emptyConfig()) {
+		assert.Fail(t, "Config should be empty, %+v\n", configObj)
+	}
```

Comment:
Not for this PR, but perhaps in the next iteration we should look into seeing if this can be thrown as an error? It would be useful to catch typos (e.g., if I mistyped `include`, I would want it to give me an error than move along as if it didn't exist!)

Reply:
I think this would be awesome. Any unparseable key in the config file should raise an informative error.